### PR TITLE
Styleguide applied to marks for new and updated

### PR DESCRIPTION
**Pull request by Graham-72**
https://github.com/backdrop/backdrop-issues/issues/760
Change CSS for class .marker

### CODE REVIEW

File: core/modules/system/css/system.theme.css [hunk 1]
```diff
@@ -113,8 +113,8 @@ input.form-radio {
   vertical-align: middle;
 }
 .marker,
-.form-required {
-  color: #f00;
+ .form-required {
+   color: #f00;
```
Comment (wesruv) [inline core/modules/system/css/system.theme.css:117]: Nitpicky, but need to get rid of these extra spaces here on line 116 and 117 before ".form-required" and "color"

File: core/themes/seven/css/seven.fonts.css [hunk 2]
```diff
@@ -14,12 +14,12 @@
   font-family: 'Open Sans';
   font-style: italic;
   font-weight: 400;
-  src: url('../fonts/OpenSans-Light-webfont.eot'); /* Old IE */
+  src: url('../fonts/OpenSans-Italic-webfont.eot'); /* Old IE */
```
Comment (wesruv) [inline core/themes/seven/css/seven.fonts.css:17]: Can't believe I goofed that up! But we should put the edits to this file in a separate PR since it's off the topic here.

File: core/misc/dropbutton.css [hunk 3]
```diff
@@ -127,7 +127,7 @@
   right: auto;
 }
 .dropbutton-multiple.open .dropbutton-arrow {
-  border-bottom: 0.3333em solid;
+  border-bottom: 0.7em solid;
```
Comment (wesruv) [inline core/misc/dropbutton.css:130]: Changes to this file should also be in a separate PR

**Comment by wesruv** [comment]
I don't see any updates for the `.marker` element in here? Little confused.

Reading the title it seems like the goal was to edit the `.marker` elements ('updated' and 'new') but I don't really see them in here.

I saw a note you wrote saying you were trying to sort out the Github stuff, if there's anything I can help with, let me know!

**Comment by wesruv** [comment]
... I just noticed that you and Nate talked about this one in Issue #878, sorry for my confusion :/

**Comment by quicksketch** [comment]
Yeah this PR is all over the place. The new PR for new and updated marks is at https://github.com/backdrop/backdrop/pull/828.
